<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DentalWidgetControls</class>
 <widget class="QWidget" name="DentalWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>599</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>图像成分提取</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>表面重建</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>阈值下限: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_surfaceLowerBound">
              <property name="text">
               <string>1500</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_useSmooth">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>平滑化</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>平滑化迭代次数: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_smoothIteration">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>50</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>迭代因子: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_smoothRelaxtion">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_reconstructSurface">
            <property name="text">
             <string>表面重建</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>术中 CT 影像: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_intraopCt" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>数据空间位置移动</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>移动或旋转数据（以给定方向平移 λ mm 或 以给定轴旋转 λ 度）</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="6">
             <widget class="QPushButton" name="pushButton_rxm">
              <property name="text">
               <string>RX-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_xp">
              <property name="text">
               <string>X+</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="pushButton_zp">
              <property name="text">
               <string>Z+</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pushButton_xm">
              <property name="text">
               <string>X-</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="pushButton_zm">
              <property name="text">
               <string>Z-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="7">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="pushButton_ym">
              <property name="text">
               <string>Y-</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="pushButton_rxp">
              <property name="text">
               <string>RX+</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="pushButton_yp">
              <property name="text">
               <string>Y+</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="6">
             <widget class="QPushButton" name="pushButton_rzm">
              <property name="text">
               <string>RZ-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QPushButton" name="pushButton_ryp">
              <property name="text">
               <string>RY+</string>
              </property>
             </widget>
            </item>
            <item row="2" column="6">
             <widget class="QPushButton" name="pushButton_rym">
              <property name="text">
               <string>RY-</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>(mm or degrees)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QPushButton" name="pushButton_rzp">
              <property name="text">
               <string>RZ+</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_intuitiveValue_1">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_14">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>λ =</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>192</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>表面配准</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>标记点配准</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>目标点群:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_LandmarkTargetPointset" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>初始点群:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_LandmarkSrcPointset" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_Landmark">
            <property name="text">
             <string>标记点配准</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>ICP 配准</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>目标点群:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_IcpTargetPointset" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>初始表面:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_IcpSrcSurface" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_Icp">
            <property name="text">
             <string>ICP配准</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>配准计算所得矩阵</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_14">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_12">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_13">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_11">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_9">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_10">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_15">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_7">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_0">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_8">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_RegistrationMatrix_5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>应用配准矩阵</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>移动对象:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QmitkSingleNodeSelectionWidget" name="mitkNodeSelectWidget_MovingObject" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ApplyRegistrationMatrix">
            <property name="text">
             <string>应用配准矩阵</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输出：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
